### ISSUE TITLE
DVM: Tutorial, blog post & short screencast

### ISSUE THREAD

**Issue by jnoller** [getcarina/getcarina.com#504]
DVM solves a Real Problem - once the cleanup is done and @rgbkrk greenlights it, we should do a tutorial, blog and screencast on it

* [X] Tutorial
* [X] Blog
* [x] Screencast

**Comment by kmsholcomb** [comment]
@carolynvs: Does this issue relate to the blog post and dvm tutorial you have written? If so, do you think we can close this issue? Or do we still need a screencast? Perhaps we could just leave a comment here about the progress that has been made? Thanks!

**Comment by jnoller** [comment]
Screencast!!!

Jesse

**Comment by carolynvs** [comment]
Does my [gif](https://twitter.com/carolynvs/status/664498031841181696) count as a screencast? :smile:   If you have an example screencast that we've already done, that would be very helpful.

**Comment by rgbkrk** [comment]
Updated the issue text with a checklist.

Do you have screenflow @carolynvs? I can also roll with this one.

**Comment by everett-toews** [comment]
@carolynvs @rgbkrk Ping. Should Kyle take the screencast?

**Comment by rgbkrk** [comment]
Yes indeedy. Doing it now.

**Comment by rgbkrk** [comment]
Here's a draft screencast:

[dvm.mp4.zip](https://github.com/getcarina/getcarina.com/files/61670/dvm.mp4.zip)

**Comment by rgbkrk** [comment]
Where are we pushing screencasts to? It's braindead simple for me to post them to my personal YouTube or an account we run. Otherwise, happy to upload them... somewhere.

**Comment by lauracly** [comment]
Did you get an answer on where to post the screencasts? Is this something that still needs to be figured out?

**Comment by rgbkrk** [comment]
We're all set now and these have been submitted to the youtube channel for upload.
